src/runtime_src/core/edge/user/shim.cpp

```diff
@@ -471,8 +471,14 @@ int ZYNQShim::xclLoadAxlf(const axlf *buffer)
 		mLogStream << __func__ << ", " << std::this_thread::get_id() << ", " << buffer << std::endl;
 	}
 
-	auto runtime_pr_en = xrt_core::config::get_pr_enable();
-	if (runtime_pr_en == true) {
+  // If platform is a non-PR-platform, Following check will fail. Dont download the partial bitstream
+  // if Platform is a PR-platform, Following check passes as enable_pr value is true by default. Download the partial bitstream
+  // If platform is a PR-platform, but v++ generated a full bitstream (using some v++ param). User need to add enable_pr=false in xrt.ini.
+
+  auto is_pr_platform = (buffer->m_header.m_mode == XCLBIN_PR ) ? true : false;
+	auto runtime_pr_en = xrt_core::config::get_enable_pr(); //default value is true
+	
+  if (is_pr_platform && runtime_pr_en) {
```

Comment:
Indentation

Reply:
> 
> 
> Indentation for this function.

Hi Min,
Fixed the indentation of whole function. This is now similar to rest of the functions